app/server/src/lib/compiler/json-to-csf-compiler.test.ts

```diff
@@ -1,13 +1,15 @@
 import 'jest-specific-snapshot';
 import path from 'path';
 import fs from 'fs-extra';
+import YAML from 'yaml';
 import { compileCsfModule } from '.';
 
-const inputRegExp = /\.json$/;
+const inputRegExp = /\.(json|ya?ml)$/;
```

Comment:
Rather than this can we organize it as two `describe` blocks so something like:

```
['json', 'yaml'].each (fileType) -> {
  const inputRegExp = new RegExp(`\.(${fileType}\$`);

  describe(`${fileType}-to-csf-compiler`, () => {
    .....
```

Reply:
It is fixed in d1be58d 🙂